libnmstate/netapplier.py

```diff
@@ -150,6 +152,13 @@ def _apply_ifaces_state(
         raise NmstatePermissionError('Error creating a check point')
     except nm.checkpoint.NMCheckPointCreationError:
         raise NmstateConflictError('Error creating a check point')
+    except NmstateError:
+        # Assume rollback occured, revert IPv6 stack state.
```

Comment:
I'm confused with the wording here; does this roll back or not ?

If not I agree whit @cathay4t comments, if so, I don't understand them.

Reply:
@maiqueb Currently, we just send a request to NM asking a rollback when failure. But we don't check on whether rollback finished. Hence @EdDev put a `sleep(5)` for that.

Once we have rollback wait support, we can remove this sleep.

Reply:
Ah ok. Got it, tks.